src/components/DateRangePicker.js

```diff
@@ -33,7 +34,7 @@ class DateRangePicker extends React.Component {
     selectedEndDate: PropTypes.number,
     selectedStartDate: PropTypes.number,
     showDefaultRanges: PropTypes.bool,
-    style: PropTypes.object
```

Comment:
@bsbeeks thank you for adding the `styles` prop 👍 
I have one more small request, can you bring back this `style` prop so this PR isn't a breaking change and to match the example that I provided?

https://github.com/mxenabled/mx-react-components/blob/master/src/components/SimpleSelect.js#L109

Reply:
Gotcha. It seemed redundant to have both but I understand the desire to avoid a breaking change.